paddle/fluid/inference/api/analysis_config.cc

```diff
@@ -212,14 +226,24 @@ void AnalysisConfig::Update() {
     }
 #ifdef PADDLE_WITH_MKLDNN
     pass_builder()->EnableMKLDNN();
-    use_mkldnn_ = true;
 #else
     LOG(ERROR) << "Please compile with MKLDNN first to use MKLDNN";
     use_mkldnn_ = false;
 #endif
   }
 
-  if (enable_memory_optim_) {
+  // Any quantization passes must come after all other optimization passes
+  if (use_quantizer_) {
+    if (!enable_ir_optim_) {
+      LOG(ERROR)
+          << "EnableQuantizer() only works when IR optimization is enabled.";
+    }
+    // TODO(wojtuss): enable below after PR #16326 is merged
+    /* pass_builder()->EnableQuantizer(); */
```

Comment:
@wojtuss #16326 is merged already. But do we need line 236-246?

Reply:
fixed in https://github.com/PaddlePaddle/Paddle/pull/16396